glib/src/subclass/object.rs

```diff
@@ -31,7 +31,7 @@ pub trait ObjectImpl: ObjectSubclass + ObjectImplExt {
     ///
     /// This is called whenever the property value of the specific subclass with the
     /// given index should be returned.
-    fn get_property(&self, _obj: &Self::Type, _id: usize) -> Result<Value, ()> {
+    fn get_property(&self, _obj: &Self::Type, _id: usize) -> Value {
```

Comment:
Need to update gstreamer-rs / gst-plugins-rs before this can be merged, otherwise things will be broken for a while :)

Reply:
I mean: I'll do that once this PR here is ready but don't merge it before that.

Reply:
37 lines to change, I'll take care of that when merging :)